Makefile

```diff
@@ -62,9 +62,13 @@ node_modules: package.json ## install node modules
 ## --- 🚀 Release management ----------------------------------------------------
 
 .PHONY: prerelease
-prerelease: ## create a new pre-release
+prerelease: ## release current changes as prerelease versions
 	@bin/make/prerelease.sh
 
+.PHONY: graduate
+graduate: ## graduate prerelease versioned packages to stable versions
```

Comment:
Does it mean only pre-release versions will be graduated and other packages will remain intact?

Reply:
Yes, only pre-release versions.